src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -170,12 +170,11 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
 
     private Errors validatePartialRemoveDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
-        // TODO Need to also perform partial validation of the common details (like for update journey). See UAR-1461.
-        //      Probably a call to validatePartialCommonDetails(overseasEntitySubmissionDto, errors, loggingContext)
+        errors = validatePartialCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
 
         var removeDto = overseasEntitySubmissionDto.getRemove();
         if (removeDto != null) {
-            removeValidator.validate(removeDto, errors, loggingContext);
+            removeValidator.validate(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
i think the removeDto is still needed on line 177 instead of overseasEntitySubmissionDto. I think the removeDto refers to the 'remove' section in the overseasEntitySubmissionDto and the removeValidator is the validator for that remove section. Maybe @mwestacott can confirm.

Reply:
I think @mattch1 's change is right actually, as the remove validator needs access to the 'higher level' object (`overseasEntitySubmissionDto`) in order to check that the filing date isn't present.

Reply:
Strike that. See comments below.

Reply:
We had a call and decided to move the date check out of the remove validator